playground/src/components/selectors/experiment_submissions_select.tsx

```diff
@@ -0,0 +1,351 @@
+            </div>
+          </div>
+        </div>
+      )}
+      <div className="flex gap-2">
+        <div className="flex-1 p-1 space-y-1">
+        {!disabled && (
+            <div className="justify-between flex items-center">
+              {trainingSubmissions !== undefined && (
+                <button
+                  disabled={
+                    Math.min(
+                      moveSubmissionsNumber,
+                      excludedSubmissions.length ?? 0
+                    ) === 0
+                  }
+                  className="rounded-md p-2 text-primary-500 hover:text-primary-600 hover:bg-primary-100 disabled:text-gray-500 disabled:cursor-not-allowed disabled:hover:bg-gray-100"
+                  onClick={() => moveSubmissions("excluded", "training")}
+                >
+                  Move{" "}
+                  {Math.min(
+                    moveSubmissionsNumber,
+                    excludedSubmissions.length ?? 0
+                  )}{" "}
+                  to Training →
+                </button>
+              )}
+              <button
+                disabled={
+                  Math.min(
+                    moveSubmissionsNumber,
+                    excludedSubmissions.length ?? 0
+                  ) === 0
+                }
+                className="rounded-md p-2 text-primary-500 hover:text-primary-600 hover:bg-primary-100 disabled:text-gray-500 disabled:cursor-not-allowed disabled:hover:bg-gray-100"
+                onClick={() => moveSubmissions("excluded", "evaluation")}
+              >
+                Move{" "}
+                {Math.min(
+                  moveSubmissionsNumber,
+                  excludedSubmissions.length ?? 0
+                )}{" "}
+                to Evaluation →
+              </button>
+            </div>
+          )}
+          <div className="text-base font-medium border-b border-gray-300 mb-2">
+            Excluded ({excludedSubmissions.length} Submissions)
+          </div>
+          <p className="text-sm text-gray-500 mb-2">
+            Submissions that are not used in the experiment.
+          </p>
+          <SubmissionList
+            submissions={excludedSubmissions}
+            feedbacks={feedbacks}
+          />
+        </div>
+        {trainingSubmissions !== undefined && (
+          <div className="flex-1 p-1 space-y-1">
+            {!disabled && (
+              <div className="justify-between flex items-center">
+                <button
+                  disabled={
+                    Math.min(
+                      moveSubmissionsNumber,
+                      trainingSubmissions.length ?? 0
+                    ) === 0
+                  }
+                  className="rounded-md p-2 text-red-500 hover:text-red-600 hover:bg-red-100 disabled:text-gray-500 disabled:cursor-not-allowed disabled:hover:bg-gray-100"
+                  onClick={() => moveSubmissions("training", "excluded")}
+                >
+                  Move{" "}
```

Comment:
```suggestion
                  ← Move{" "}
```
I'd put the arrow in front

Reply:
Done